src/Model/BaseBlock.php

```diff
@@ -160,11 +160,11 @@ public function getUpdatedAt()
         return $this->updatedAt;
     }
 
-    public function addChildren(BlockInterface $child)
+    public function addChildren(BlockInterface $children)
```

Comment:
We should rename the method to `addChild` in next major.

Reply:
Yes 👍 the implementation makes very little sense with this naming.